cpp/src/slice2py/PythonUtil.cpp

```diff
@@ -1128,38 +1165,29 @@ Slice::Python::CodeVisitor::visitClassDefStart(const ClassDefPtr& p)
     _codeFragments.push_back(createCodeFragmentForPythonModule(p->declaration(), outF.str()));
 
     // Emit the class definition.
-    BufferedOutput out;
+    _out = std::make_unique<BufferedOutput>();
+    auto& out = *_out;
```

Comment:
Would be good to find a better pattern, maybe keep the output in the fragment. And have createFragment/currentFragment like:

```
auto out = createFragment(p);

out = currentFragment(p);
```